dictionary_client/commands.py

```diff
@@ -32,11 +34,13 @@ def disconnect_command():
 
 @dictionary_server_command
 def define_word_command(word, db="*"):
-    return f"DEFINE {db} {word}"
+    word = format_word(word)
+    return f"Define {db} {word}"
```

Comment:
For consistency, can we keep the command "DEFINE" in uppercase? e.g. https://datatracker.ietf.org/doc/html/rfc2229#section-3.2

Reply:
Oh that's a typo, I'll fix it!